subsys/bluetooth/host/classic/sco.c

```diff
@@ -403,3 +442,27 @@ struct bt_conn *bt_conn_create_sco(const bt_addr_t *peer, struct bt_sco_chan *ch
 
 	return sco_conn;
 }
+
+int bt_sco_conn_cb_register(struct bt_sco_conn_cb *cb)
+{
+	if (sys_slist_find(&sco_conn_cbs, &cb->_node, NULL)) {
+		return -EEXIST;
+	}
+
+	sys_slist_append(&sco_conn_cbs, &cb->_node);
+
+	return 0;
+}
+
+int bt_sco_conn_cb_unregister(struct bt_sco_conn_cb *cb)
+{
+	CHECKIF(cb == NULL) {
+		return -EINVAL;
+	}
```

Comment:
Any reason you're doing a NULL check for `unregister()` but not for `register()`?